ProcessLib/HeatTransportBHE/HeatTransportBHEProcess.cpp

```diff
@@ -288,18 +288,14 @@ void HeatTransportBHEProcess::createBHEBoundaryConditionTopBottom(
         // the number of connected line elements.
         for (auto const& bhe_node : bhe_nodes)
         {
-            auto const connected_elems = bhe_node->getElements();
-
-            int cnt = 0;
-
-            for (auto const element : connected_elems)
-            {
-                if (element->getDimension() == 1)
-                {
-                    cnt++;
-                }
-            }
-            if (cnt == 1)
+            // Count number of 1d elements in every BHE node.
+            const std::size_t nElems = std::count_if(
```

Comment:
```suggestion
            const std::size_t n_elements = std::count_if(
```
At least the camel-case should be fixed.

Reply:
Changed.